typescript/test/test_application_js/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "scripts": {
     "test": "bash showcase-setup.sh",
-    "browser-test": "karma start"
+    "browser-test": "showcase-server/gapic-showcase run & karma start"
```

Comment:
Let's just do `karma start` here. Starting showcase server is a separate task that should be done elsewhere (e.g. in CircleCI config).

Reply:
cool, will make changes!